lfs/util.go

```diff
@@ -326,3 +327,39 @@ func FileExistsOfSize(path string, sz int64) bool {
 
 	return !fi.IsDir() && fi.Size() == sz
 }
+
+func CopyFileContents(src string, dst string) error {
+	tmp, err := ioutil.TempFile(TempDir, filepath.Base(dst))
+	if err != nil {
+		return err
+	}
+	defer func() {
+		tmp.Close()
+		_ = os.Remove(tmp.Name())
+	}()
+	in, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+	defer in.Close()
+	_, err = io.Copy(tmp, in)
+	if err != nil {
+		return err
+	}
+	err = tmp.Sync()
```

Comment:
Why `Sync()` instead of `Close()`? Seems like you'd want `Sync()` in cases where you want to flush to disk while keeping the handle open. Seems like we just want to close it here.